src/modules/10-common/utils/YamlHelperMethods.ts

```diff
@@ -5,21 +5,24 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import { Document } from 'yaml'
-import type { Options } from 'yaml'
-import { load } from 'js-yaml'
+/* disable lint in this file as we are exporting wrappers */
+
+// eslint-disable-next-line no-restricted-imports
+import type { DumpOptions } from 'js-yaml'
+// eslint-disable-next-line no-restricted-imports
+import { load, dump } from 'js-yaml'
```

Comment:
Could these be combined?
```suggestion
import { load, dump, DumpOptions } from 'js-yaml'
```

Reply:
I prefer to keep type imports separate for better readability

Reply:
Not a bad technique. I think our Sonar setup will complain, but it's a decent use-case 👍🏻